app/main/views/briefs.py

```diff
@@ -116,13 +116,16 @@ def brief_response(brief_id):
 
     if 'areaOfExpertise' in brief:
         current_supplier = data_api_client.req.suppliers(current_user.supplier_code).get()
-
-        if not supplier_is_assessed(current_supplier, brief['areaOfExpertise']):
-            if supplier_is_unassessed(current_supplier, brief['areaOfExpertise']):
-                return redirect(url_for(".create_assessment", brief_id=brief_id))
-            else:
-                current_domain = data_api_client.req.domain(brief['areaOfExpertise']).get()
-                return redirect('/case-study/create/{}/brief/{}'.format(current_domain['domain']['id'], brief_id))
+        outcome_but_has_assessed_domains = (brief['lotSlug'] == 'digital-outcome' and
+                                            len(current_supplier['supplier']['domains'].get('assessed', [])) > 0)
+
+        if not outcome_but_has_assessed_domains and \
+                not supplier_is_assessed(current_supplier, brief['areaOfExpertise']):
```

Comment:
very confusing with the 
`not supplier_is_assessed(`
`supplier_is_unassessed`(